mne/event.py

```diff
@@ -827,3 +828,289 @@ def concatenate_events(events, first_samps, last_samps):
+        s += 'delay: %.3f (s)>' % self.delay
+        return s
+
+
+class Elekta_category(object):
+    """ Represents an averaging category in Elekta/Neuromag system. """
+
+    # original DACQ variable names
+    vars = ['Comment', 'Display', 'Start', 'State', 'End', 'Event',
+            'Nave', 'ReqEvent', 'ReqWhen', 'ReqWithin',  'SubAve']
+
+    def __init__(self, comment, display, start, state, end, event, nave,
+                 reqevent, reqwhen, reqwithin, subave):
+        self.comment = comment
+        self.display = True if display == u'1' else False  # show online
+        self.state = True if state == u'1' else False  # enabled or not
+        self.start = float(start)  # epoch start
+        self.end = float(end)  # epoch end
+        self.nave = int(nave)  # desired n of averages
+        self.event = int(event)  # the reference event (index to event list)
+        self.reqevent = int(reqevent)  # required event (index to event list)
+        self.reqwhen = int(reqwhen)  # 1=before, 2=after ref. event
+        # time window before or after ref. event (s)
+        self.reqwithin = float(reqwithin)
+        self.subave = int(subave)  # subaverage size
+        # non-DACQ vars
+        # list of t=0 times (in samples) for corresponding epochs
+        self.times = None
+        # index of category in DACQ list
+        self.index = None
+
+    def __repr__(self):
+        s = '<Elekta_category | '
+        s += '"%s", ' % self.comment
+        s += 'ref. event: %d, ' % self.event
+        if self.reqevent:
+            s += 'req. event: %d ' % self.reqevent
+            s += 'within: %.3f (s) ' % self.reqwithin
+            s += 'before ' if self.reqwhen == 1 else 'after '
+            s += 'ref. event, '
+        s += 'start: %.3f (s), ' % self.start
+        s += 'end: %.3f (s)>' % self.end
+        return s
+
+    def epochs(self, raw, picks=None, reject=None,
+               baseline=(None, 0), stim_channel=None, mask=0):
+        """ Get mne.Epochs instance corresponding to the category. """
+        return self._parent._get_epochs(raw, self, picks=picks,
+                                        reject=reject, baseline=baseline,
+                                        stim_channel=stim_channel,
+                                        mask=mask)
+
+
+class Elekta_averager(object):
+    """ Represents averager settings of Elekta TRIUX/Vectorview systems.
+    """
+
+    # these are DACQ averager related variable names without preceding 'ERF'
+    vars = ['magMax', 'magMin', 'magNoise', 'magSlope', 'magSpike', 'megMax',
+            'megMin', 'megNoise', 'megSlope', 'megSpike', 'eegMax', 'eegMin',
+            'eegNoise', 'eegSlope', 'eegSpike', 'eogMax', 'ecgMax', 'ncateg',
+            'nevent', 'stimSource', 'triggerMap', 'update', 'version',
+            'artefIgnore', 'averUpdate']
+
+    # DACQ variable names start with one of these
+    acq_var_magic = ['ERF', 'DEF', 'ACQ', 'TCP']
+
+    def __init__(self, acq_pars):
```

Comment:
Actually maybe it should take an instance of `Raw` since 99% of the time people will want to eventually get `Epochs` from it. I guess you could make it take `Info` or `Raw`, where if it's `Raw` it gets saved for later so one could just get Epochs directly like `epochs = ElektaAverager(raw).get_epochs(condition)`